src/main/java/uk/gov/companieshouse/customerfeedbackapi/controller/HealthCheckController.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.companieshouse.customerfeedbackapi.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HealthCheckController {
+
+  @GetMapping("/customer-feedback/healthcheck")
+  public String healthcheck() {
+    return "Customer Feedback API Service is healthy";
```

Comment:
Should this explicitly return a 200 response?

Reply:
That's the default but I've made it explicit now